citesphere/src/main/webapp/WEB-INF/views/auth/group/item.html

```diff
@@ -157,14 +158,15 @@ <h2>
 					 (<th:block th:each="aff,affStatus : ${editor.affiliations}">
 					 	[[${aff.name}]]<span th:if="${!affStatus.last}">, </span>
 					 </th:block>)
-					 <span th:if="${!status.last}">; </span>
+					 <!-- <span th:if="${!status.last}">; </span> -->
```

Comment:
remove if not needed

Reply:
I have checked the other contributors and they have the separating semicolon